build.gradle

```diff
@@ -111,10 +111,14 @@ dependencies {
     compile "org.hibernate:hibernate-envers"
     compile "org.hibernate.validator:hibernate-validator"
     compile group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.3'
+    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: '5.0.1'
+    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '5.0.1'
```

Comment:
3.0.1 passes, 5.0.1 fails for stitching api and NPA

Reply:
seems to work for me